Loads/ILoads/IAreaForce.cs

```diff
@@ -0,0 +1,12 @@
+using OasysUnits;
+
+namespace MagmaWorks.Taxonomy.Loads
+{
+    public interface IAreaForce : ILoad
+    {
+        Pressure X { get; }
```

Comment:
Is it really pressure?

Reply:
It's `ForcePerArea` but we don't have that in `OasysUnits` other than `PressureUnit`, right?

Reply:
True, we use `PressureUnit`